x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -484,7 +484,9 @@ export const ConfigureCases: React.FC = React.memo(() => {
         }
         onCloseFlyout={onCloseCustomFieldFlyout}
         onSaveField={onCustomFieldSave}
-        renderHeader={() => <span>{i18n.ADD_CUSTOM_FIELD}</span>}
+        renderHeader={() => (
+          <span>{!customFieldToEdit ? i18n.ADD_CUSTOM_FIELD : i18n.EDIT_CUSTOM_FIELD} </span>
```

Comment:
in general, use the truthy approach first 

``{customFieldToEdit ? i18n.EDIT_CUSTOM_FIELD : i18n.ADD_CUSTOM_FIELD}`` is easier to read

Reply:
I've changed it, thanks :)